client/src/components/PledgeOption.jsx

```diff
@@ -0,0 +1,27 @@
+const React = require('react');
+
+const PledgeOption = (props) => {
+  let location = 'Only certain countries';
+  if (props.option.shippingLocation) {
+    location = 'Anywhere in the world'
+  }
+
+  return (
+    <div className="pledge-option">
+      <h3>Pledge ${props.option.tier} or more</h3>
+      <h4>{props.option.reward}</h4>
+      <p>[reward-desc]</p>
```

Comment:
This isn't a technical issue, but this doesn't look like clean text to render to the client.

Reply:
Can you clarify what doesn't seem clean about it? The "[reward-desc]" is just placeholder text for now.